### PR TITLE
Add utils for persisted state migration from old extension

### PR DESCRIPTION
**Pull request by lukaw3d**
Functions for migrating state from https://github.com/oasisprotocol/oasis-wallet-ext

### CODE REVIEW

File: src/utils/walletExtensionV0.ts
Comment (lubej) [inline src/utils/walletExtensionV0.ts:22]: Would be nice to have type as an enum, as it can easily be mistyped.

**Review by lubej — APPROVED** [review]
Overall looks good to me, would wait for @buberdds approval, as he is more familiar with the codebase.

**Review by buberdds — APPROVED** [review]

**Pull request merged**
